pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -80,10 +79,7 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctrl.
 	}
```

Comment:
We want to check the new status condition in here. QueryReady is not true, then we exit and perform as a no-op.